pkgs/development/tools/rust/rust-analyzer/default.nix

```diff
@@ -17,9 +16,14 @@ rustPlatform.buildRustPackage rec {
     sha256 = "sha256-6Tbgy77Essi3Hyd5kdJ7JJbx7RuFZQWURfRrpScvPPQ=";
   };
 
+  patches = [
```

Comment:
I would perhaps include a comment saying why we want to ignore those tests.
A simple "We do not care about code formatting and git tests in building a package".
It sounds obvious, however I think it makes understanding easier.
If you feel strongly against this. Feel free to ignore.